celery/backends/couchbase.py

```diff
@@ -7,6 +7,10 @@
 from celery.exceptions import ImproperlyConfigured
 from .base import KeyValueStoreBackend
 try:
+    import couchbase_ffi # noqa
```

Comment:
Shouldn't this [dependency](https://github.com/couchbaselabs/couchbase-python-cffi) be installed on the PyPy Travis CI jobs as well or is it not required?

Reply:
We don't need this dependency for the tests. The client is mocked. It is required to install the default client because otherwise we won't be able to instantiate the couchbase results backend at all.